alerter/test/alerter/alerters/node/test_cosmos.py

```diff
@@ -165,6 +167,7 @@ def setUp(self) -> None:
                         'node_id': self.test_node_id,
                         'node_parent_id': self.test_parent_id,
                         'last_monitored': self.test_last_monitored,
+                        'is_mev_tendermint_node': False,
```

Comment:
It is recommended that instead of explicitly defining the value `False` here, we define a variable called `self.test_is_mev_tendermint_node = False`, similar to how the other values are defined.

Reply:
Same thing for the rest of the tests...